README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
```

Comment:
The problem with putting it in the decorators section above is that the reader hasn't learned about Providers yet. So the logical structure I was going for is: (1) learn about Connector and (1a) learn about alternate syntax (`@connect`), then (2) learn about Provider and (2a) learn about alternative syntax (`@provide`).

I can see it would make sense to group the decorators together in an API docs page, but for this initial guide I think we want to group these sections by the concept that the reader is learning instead of grouping by syntax.